src/routes/admin/create_election/+page.svelte

```diff
@@ -0,0 +1,361 @@
+<script lang="ts">
+  import type { EventHandler } from 'svelte/elements';
+  import short from 'short-uuid';
+  import adminApi from '$lib/utils/adminApi';
+  import activate from '$lib/assets/activate.mp3';
+  import { ElectionSystems } from '$backend/types/types';
+  import { ResponseResult } from '$lib/utils/callApi';
+  import { alerts } from '$lib/stores';
+  import { goto } from '$app/navigation';
+  import { onMount } from 'svelte';
+  import { page } from '$app/stores';
+
+  type FormAlternative = { description: string; id: string };
+
+  const exisitngElectionString = $page.url.searchParams.get('election');
+  const exisitingElection = JSON.parse(exisitngElectionString);
+
+  let title = exisitingElection?.title ?? null;
+  let description = exisitingElection?.description ?? '';
+  let seats = exisitingElection?.seats ?? 1;
+  let useQualifiedMajority = exisitingElection?.useStrict ?? false;
+  let electionSystem = exisitingElection?.type ?? ElectionSystems.NORMAL;
+  let electionLocation = exisitingElection
```

Comment:
What is going on here? Is there an entire election in the search params?

Reply:
Yes, used when you copy an existing election:-)